lib/components/user/saved-trip-list.js

```diff
@@ -72,17 +69,53 @@ export default withLoggedInUserSupport(
   true
 )
 
+const TripHeader = styled.h3`
+  margin-top: 0px;
+`
+
+const PanelHeading = styled(Panel.Heading)`
+  background-color: white!important;
```

Comment:
Did you mean `background: none!important;`?

Reply:
No, I think these should be white.

Reply:
Why this specific color?

Reply:
I changed it to white to match the mockups.